ratatool-sampling/src/main/scala/com/spotify/ratatool/samplers/BigSamplerAvro.scala

```diff
@@ -60,9 +62,10 @@ private[samplers] object BigSamplerAvro {
             v.asInstanceOf[GenericRecord],
             subfields.tail.mkString(BigSampler.fieldSep.toString),
             hasher)
-        case Type.ENUM => hasher.putString(v.asInstanceOf[Enum[_]].name, BigSampler.utf8Charset)
+        case Type.ENUM =>
+          hasher.putString(v.asInstanceOf[EnumSymbol].toString, BigSampler.utf8Charset)
```

Comment:
For my own learning, what is the difference here? Avoids a casting error?

Reply:
The problem was that you got an EnumSymbol not an Enum and the program failed with an exception. Same happened for Bytes, where I got a ByteBuffer instead of Array[Byte]. Did this work before? :-)